jetty-websocket/jetty-websocket-common/src/main/java/org/eclipse/jetty/websocket/common/JettyWebSocketFrameHandler.java

```diff
@@ -211,7 +211,7 @@ public void onFrame(Frame frame, Callback callback)
         switch (frame.getOpCode())
         {
             case OpCode.CLOSE:
-                onCloseFrame(frame, demandingCallback);
+                onCloseFrame(frame, callback);
```

Comment:
added this change in the merge, small bug from #3504
if we receive a close frame we don't want to further demand we just succeed the callback

I didn't think it deserved a PR of its own so I put it here